about.html

```diff
@@ -0,0 +1,161 @@
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/featuredCourse.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <link rel="stylesheet" href="css/animations.css" />
+    <title>The Code Chaser</title>
+  </head>
+  <body>
+    <!-- #### header-section #### -->
+    <header>
+        <div class="header">
+          <a class="header-menu" href="#"><i class="fab fa-facebook-f"></i></a>
+          <a class="header-menu" href="#"><i class="fab fa-twitter"></i></a>
+          <a class="header-menu" href="#">English</a>
+          <a class="header-menu" href="#">My Page</a>
+          <a class="header-menu" href="#">Logout</a>
+        </div>
+        <div class="header2">
+      <a href="index.html" class="header-logo-a"><img src="images/header-logo.png" alt="header logo" class="header-logo" /></a>
+
+      <!-- #### navbar-section #### -->
+      <nav>
+        <div class="desktop-nav">
+          <ul class="desktop-menu-items">
+            <li>
+            <a class="desktop-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="desktop-menu-item" href="#partners">Sponser</a>
+             </li>
+             <li>
+            <a class="desktop-menu-item" href="#contact">Contact</a>
+            </li>
+            <li>
+            <a class="desktop-menu-item1" href="#">CC Campaign</a>
+            </li>
+          </ul>
+        </div>
+        <div class="mobile-nav">
+     <button type="button" class="btn-open" ><i class="fas fa-bars"></i></button>  
+        <div class="mobile-menu-container">
+              <button type="button" class="btn-close">X</button>
+          <ul class="mobile-menu-items">
+            <li>
+            <a class="mobile-menu-item" href="index.html">Home</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Program</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#">Tutors</a>
+            </li>
+             <li>
+            <a class="mobile-menu-item" href="#partners">Sponser</a>
+            </li>
+            <li>
+            <a class="mobile-menu-item" href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        </div>
+      </nav>
+      </div>
+    </header>
+
+    <!-- #### main-section #### -->
+    <main>
+
+      <!-- #### hero-section #### -->
+      <section class="hero-section">
+        <h2 class="headline-heading1">"Hello! technology geeks"</h2>
+        <h1 class="headline-heading2">
+          BECOME A PROFESSIONAL REMOTE WEB DEVELOPER
```

Comment:
- This heading should be much bigger than what it is right now. This was already pointed out by the previous reviewer and please also align the contents of the hero section to the left. Right now it appears centered on desktop screens.